src/io/linux.zig

```diff
@@ -1662,7 +1662,10 @@ pub const IO = struct {
     fn fs_supports_direct_io(dir_fd: fd_t) !bool {
         if (!@hasField(posix.O, "DIRECT")) return false;
 
-        const path = "fs_supports_direct_io";
+        var cookie: [16]u8 = .{'0'} ** 16;
+        _ = stdx.array_print(16, &cookie, "{0x}", .{std.crypto.random.int(u64)});
+
+        const path: [:0]const u8 = "fs_supports_direct_io-" ++ cookie ++ "";
```

Comment:
TIL you can use `++` at runtime with arrays!

Reply:
Yup! One of the coolest zig features!